pvpn-cli/killswitch.conf

```
@@ -0,0 +1,33 @@
+#!/bin/env net -f
```

I haven't used `nft` like this before, but this line should look like below, right?

```suggestion
#!/bin/env nft -f
```